packages/storage-service/package.json

```
@@ -0,0 +1,73 @@
+{
+  "name": "@metamask/storage-service",
+  "version": "1.0.0",
+  "description": "Platform-agnostic service for storing large, infrequently accessed controller data",
+  "keywords": [
+    "MetaMask",
+    "Ethereum",
+    "Storage",
+    "Service"
+  ],
+  "homepage": "https://github.com/MetaMask/core/tree/main/packages/storage-service#readme",
+  "bugs": {
+    "url": "https://github.com/MetaMask/core/issues"
+  },
+  "repository": {
+    "type": "git",
+    "url": "https://github.com/MetaMask/core.git"
+  },
+  "license": "MIT",
```

This field needs an update to reference both licenses as well:

```suggestion
  "license": "(MIT OR Apache-2.0)",
```

I noticed this is wrong in the example I cited too 😅 I'll fix this field for the analytics controller in a separate PR